### ISSUE TITLE
[definition] congenital renal artery stenosis

### ISSUE THREAD

**Issue by nicolevasilevsky** [monarch-initiative/mondo#1741]
**Mondo term (ID and Label)**
MONDO_0019993 congenital renal artery stenosis

**Reason for deprecation**
this seems like a phenotype and not a disease.
@pnrobinson would this be appropriate to add to HPO?

**Suggested term to consider**
HP:0001920 'Renal artery stenosis'
HP:0003577 'Congenital onset'

**Comment by pnrobinson** [comment]
I think we can use this term plus the AgeOfOnset 'Congenital'
https://hpo.jax.org/app/browse/term/HP:0001920

Does this work?

Where is MONDO_0019993 congenital renal artery stenosis coming from?

**Comment by nicolevasilevsky** [comment]
> Where is MONDO_0019993 congenital renal artery stenosis coming from?

Orphanet:97598

**Comment by nicolevasilevsky** [comment]
@pnrobinson I'll add this new term to HPO, thanks!

**Comment by pnrobinson** [comment]
Don't understand -- there already is a term? We can use that and if we need to say congenital we can use a modifier. That is exceedingly rare afaik.

**Comment by nicolevasilevsky** [comment]
Oh! I see what you mean. I'll close the PR to add the new term then. Thanks for clarifying!

**Comment by nicolevasilevsky** [comment]
we should not obsolete this - this IS a disease.

@pnrobinson 
you offered to look for a PMID, thank you! :)

**Comment by nicolevasilevsky** [comment]
We need to add a definition
